.github/workflows/docs.yml

```diff
@@ -25,7 +25,14 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
+
```

Comment:
Consider removing this empty line.

Reply:
I added this for clarity/consistency as all other steps have a blank line between them.